api/user/relations.py

```diff
@@ -13,3 +13,9 @@
     db.Column("author_id", db.Integer, db.ForeignKey("author.id"), primary_key=True),
     db.Column("post_id", db.Integer, db.ForeignKey("post.id"), primary_key=True),
 )
+
+post_images = db.Table(
+    "post_images",
+    db.Column("post_id", db.String(50), db.ForeignKey("post.id")),
+    db.Column("image", db.Text, nullable=False),
+)
```

Comment:
I'm not sure this is how hindle wants us to implement these, I thought he wanted us to implement images specifically as a unlisted post, base64 encoded?

Reply:
This is something that needs clarification. If we allow both images and captions to be part of posts, hindle said we can have 2 separate posts for them. Thing is how would backend know that the post is not just text but images since the image would have a spearate id then and we need some way to tell that a post has a post associated with it.